packages/eslint-plugin/docs/rules/README.md

```diff
@@ -6,18 +6,27 @@ pagination_prev: null
 slug: /
 ---
 
-`@typescript-eslint/eslint-plugin` includes over 100 rules that detect best practice violations, bugs, and/or stylistic issues specifically for TypeScript code.
-See [Configs](/linting/configs) for how to enable recommended rules using configs.
+## Introduction
```

Comment:
H2s called "Introduction" are not customary in our docs. I suggest removing it, since the first paragraph is introduction by default.